### PR TITLE
Update react-native from 0.62.2 to 0.63.3.

### PR DESCRIPTION
**Pull request by yatatsu**
Update React Native to latest version.

- https://react-native-community.github.io/upgrade-helper/?from=0.62.2&to=0.63.3

## Summary

- Update dependencies
- Update example
  - Update Podfile
  - Add LaunchScreen.storyboard
  - Delete LaunchScreen.xib
  - Update gradle version
  - Update build.gradle
- Update travis ci setting
  - Add snapshot and remove jest's option [`--updateSnapshot`](https://jestjs.io/docs/ja/snapshot-testing)
  - Fix cache directory
  - Update android build setting
  - Use `yarn --frozen-lockfile` instead of `yarn install`
- **Breaking Change!!** Change iOS style setting.
  - All color property types turns into `ColorValue`.
  - https://reactnative.dev/docs/colors

## ColorValue example

- No need to call `processColor()`.
- `RgbaColor` type is removed.

```diff
const iOSCardFormStyle: IOSCardFormStyle = {
-    labelTextColor: {
-        r: 0,
-        g: 0.4,
-        b: 0.8,
-    },
-    inputTextColor: processColor("#004488"),
-    submitButtonColor: processColor("#0055ff"),
+    labelTextColor: PlatformColor("label"),
+    inputTextColor: "rgba(255, 0, 255, 1.0)",
+    submitButtonColor: "#0055ff",
}
```

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,43 +1,56 @@
...
     node_js: 10
+    os: linux
+    dist: trusty
+    cache:
+      yarn: true
+      directories:
+        - node_modules
+        - example/node_modules
+    install:
+      - yarn --frozen-lockfile
```
Comment (darai0512) [inline .travis.yml:17]: yarn.lockをcommit管理しているので、異なるinstallになったら落ちる設定に

**Comment by darai0512** [comment]
文字ベースのchangelog
color周りの変更を確認
https://github.com/react-native-community/releases/blob/master/CHANGELOG.md

File: example/android/app/build.gradle [hunk 2]
```diff
@@ -161,20 +161,13 @@ android {
         }
         release {
             // Caution! In production, you need to generate your own keystore file.
-            // see https://facebook.github.io/react-native/docs/signed-apk-android.
+            // see https://reactnative.dev/docs/signed-apk-android.
```
Comment (darai0512) [inline example/android/app/build.gradle:164]: 直すほどじゃないですが最後のピリオドない方が嬉しそう

File: example/ios/Podfile [hunk 3]
```diff
@@ -1,78 +1,12 @@
-platform :ios, '10.0'
+require_relative '../node_modules/react-native/scripts/react_native_pods'
```
Comment (darai0512) [inline example/ios/Podfile:1]: すっきり

File: src/CardForm.ts [hunk 4]
```diff
@@ -23,23 +23,13 @@ export type OnCardFormProducedToken = (token: Token) => void;
...
-    g: number;
-    b: number;
-    a?: number;
+    labelTextColor?: ColorValue;
+    inputTextColor?: ColorValue;
+    errorTextColor?: ColorValue;
+    tintColor?: ColorValue;
+    inputFieldBackgroundColor?: ColorValue;
+    submitButtonColor?: ColorValue;
+    highlightColor?: ColorValue;
```
Comment (darai0512) [inline src/CardForm.ts:32]: すっきり

**Review by darai0512 — APPROVED** [review]

File: example/android/app/build.gradle (hunk 2 above)
Comment (yatatsu) [reply example/android/app/build.gradle:164]: 確かに... templateを踏襲しているので一旦そのままにします。

**Pull request merged**